packages/web-app/pages/agreements/index.tsx

```diff
@@ -23,41 +20,38 @@ export const getStaticProps: GetStaticProps = async ({ locale }) => {
 
 const Agreements = () => {
 	const router = useRouter();
-	const { subgraphURI } = useConstants();
 
 	return (
-		<ApolloProvider client={client(subgraphURI ?? defaultURI ?? "")}>
-			<AgreementListProvider>
-				<article className="w-full flex justify-center">
-					<div className="absolute top h-60 w-full bg-pr-c-green1 z-1"></div>
-					<div
-						id="agreementsPage"
+		<AgreementListProvider>
+			<article className="w-full flex justify-center">
```

Comment:
This page no longer exist with this name, if you can change it to agreements.tsx directy on root, will be better to handle conflicts